server/index.test.js

```diff
@@ -0,0 +1,10 @@
+const request = require('supertest');
```

Comment:
Might be a good idea to make a directory for your tests that's separate from your other files. It'll make it easier for you when you start having more files to look through